### PR TITLE
Add vagrant tests for add/remove checks with sensu::check

### LINKED ISSUE #747: Add a test in vagrant for PR #745
Show an example of a sensu::check with ensure => false

### PR DESCRIPTION
**Pull request by Phil-Friderici**
Welcome my first tests for Vagrant. Hope this is what was indented.

`add_remove-check.sh` will test if the check `check_to_remove` is available. Depending of the result, the check will get created or deleted.

The script is prepared for further automation and gives exit code `0` only if successfully.

Fixes #747

### CODE REVIEW

File: tests/add_remove-check.pp [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    client_deregister     => true,
+    client_deregistration => {'handler' => 'deregister_client'},
+  }
+
+  if $::test == 'add' {
+    sensu::check { 'check_to_remove':
+      command        => 'PATH=$PATH:/usr/lib64/nagios/plugins check_ntp_time -H :::address::: -w 30 -c 60',
+      standalone     => false,
+      handlers       => 'default',
+      subscribers    => 'roundrobin:poller',
```
Comment (ghoneycutt) [inline tests/add_remove-check.pp]: travis is complaining about the indentation here

**Comment by ghoneycutt** [comment]
Great work!

File: tests/add_remove-check.pp (hunk 1 above)
Comment (Phil-Friderici) [reply tests/add_remove-check.pp]: should be fixed now

**Comment by ghoneycutt** [comment]
Released in v2.36.0

**Pull request merged**